packages/playground/src/types/index.ts

```diff
@@ -12,12 +12,19 @@ export interface K8SWorker {
   planetary: boolean;
   rootFsSize: number;
   farm?: Farm;
+  selectedNode: any;
+  rentedBy?: number;
+  dedicated: boolean;
+  certified: boolean;
 }
 
 export interface CaproverWorker {
+  selectedNode?: any;
```

Comment:
as i mentioned in `K8SWorker` if we can avoid using any will be much better

Reply:
also done.